src/ebur128.rs

```diff
@@ -719,6 +779,13 @@ impl EbuR128 {
     }
 
     /// Get global integrated loudness in LUFS across multiple instances.
+    ///
+    /// This can be used to allow parallel iteration of long signals, assuming some care is taken:
+    ///  1. Divide input-signal up in "chunks" of even 100ms samples. Make chunks overlap by 400ms, for example (0-10s, 9.6-20s, 19.6-30s, ...)
+    ///  2. For each chunk, create a new instance of EbuR128, then in parallel:
+    ///     1. Feed the first 100ms of the chunk through seed_frames_* function. This is sufficient to make filter-states in each instance what they would have been if a single analyzer would have reached this point.
```

Comment:
```suggestion
    ///     1. Feed the first 100ms of the chunk through `seed_frames_*` function. This is sufficient to make filter-states in each instance what they would have been if a single analyzer would have reached this point.
```

Reply:
Also that shouldn't be the first 100ms of each chunk, but the last 100ms of the previous chunk or not?

Reply:
It does not really matter. Technically, the difference is whether the chunks will be `[0..10, 9.6..20]` or `[0..10.4, 10..20.4]`, in either case, the first 400 ms will be "the overlap with previous chunk". Maybe I can make it more clear in a parenthesis though.